scripts/linters/general_purpose_linter.py

```diff
@@ -23,13 +23,8 @@
 
 from typing import Dict, Final, List, Pattern, Tuple, TypedDict
 
-from . import js_ts_linter
-from . import linter_utils
-from . import warranted_angular_security_bypasses
-
-from .. import build
-from .. import common
-from .. import concurrent_task_utils
+from .. import build, common, concurrent_task_utils
+from . import js_ts_linter, linter_utils, warranted_angular_security_bypasses
 
 MYPY = False
 if MYPY:  # pragma: no cover
```

Comment:
@Helper2020 Let's also do the forbidding of `#\s*fmt:\s*skip` in this PR. Could you add an entry in BAD_PATTERNS in this file to prevent that, and a test as well?